### PR TITLE
9.get/api/articles/article id/comments

### PR DESCRIPTION
**Pull request by garysum88**
Completed ticket #9 GET /api/articles/:article_id/comments

1. Updated app.js
2. Updated article controller for getComments func
3. Updated article model for fetchComments func
4. Updated app.test.js to introduce 4 news for this endpoint

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -233,4 +233,61 @@ describe("8. GET /api/articles", () => {
...
+                            created_at : expect.any(String),
+                            author : expect.any(String),
+                            body : expect.any(String)
+                        })
+                    )
+                })
+            })
+        })
+    
+        test("returns status 404 with passed an article id which exists but has no associated comment", () => {
```
Comment (popatre) [inline __tests__/app.test.js:261]: This would be a 200. 

The request has happened, but has no comments, so just returns an empty array

Comment (popatre) [reply __tests__/app.test.js:261]: 404 would suggest some kind of error

File: models/article.model.js [hunk 2]
```diff
@@ -64,3 +64,31 @@ exports.fetchAllArticles = () => {
...
+
+    const checkArticleIdPromise =
+    db.query(`SELECT * FROM articles WHERE article_id = $1`,[articleId])
+
+    const fetchCommentsPromise = db.query('SELECT comment_id, votes, created_at, author, body FROM comments WHERE article_id = $1',[articleId])
+    
+    return Promise.all([checkArticleIdPromise,fetchCommentsPromise])
+    .then((resolvedArr)=> {
+
+        // console.log(resolvedArr[0].rows,"check")
```
Comment (popatre) [inline models/article.model.js:78]: Remove console logs

**Review by popatre — CHANGES_REQUESTED** [review]
Just need that 404 changed to 200 with an empty array

File: models/article.model.js (hunk 2 above)
Comment (garysum88) [reply models/article.model.js]: done

File: __tests__/app.test.js (hunk 1 above)
Comment (garysum88) [reply __tests__/app.test.js]: got it. I have further updated the article model

**Pull request merged**